hooks/apply-git-secrets.sh

```diff
@@ -13,8 +13,6 @@ IFS=$'\n\t'
 } || {
   :
 }
-git secrets --add 'private_key'
-git secrets --add 'private_key_id'
-git secrets --add --allowed --literal "git secrets --add 'private_key'"
-git secrets --add --allowed --literal "git secrets --add 'private_key_id'"
-git secrets --add --allowed --literal 'tmp_private_key = `grep private_key_id #{creds_file.path} | cut -d\\\" -f4`.strip()'
+git secrets --add '"private_key":'
+git secrets --add '"private_key_id":'
```

Comment:
My sense is that a `private_key_id` is actually not sensitive, so maybe we should constrain this search to `private_key` only?